schemas/aev1_frame_assembled_event_v1.fbs

```diff
@@ -0,0 +1,13 @@
+include "frame_metadata_v1.fbs";
```

Comment:
Do not include the schemas here, the single source of truth for those is https://github.com/STFC-ICD-Research-and-Design/supermusr-data-pipeline